src/App.tsx

```diff
@@ -36,14 +29,7 @@ const App: React.FC = () => (
   <ErrorBoundary>
     <ContextProviders>
       <Router>
-        <GQLWrapper
-          gqlURL={getGQLUrl()}
-          isDevelopment={isDevelopment()}
-          isTest={isTest()}
-          schemaString={getSchemaString()}
-          credentials="include"
```

Comment:
I think we may need this prop set to pass along the cookie in our GQL requests

Reply:
yup, that's why it's still included